3rd-party/vaultpress.php

```diff
@@ -22,8 +24,8 @@ function jetpack_vaultpress_rewind_enabled_notice() {
 			<?php
 				echo sprintf(
 					wp_kses(
-						/* Translators: first variable is the URL of the web site without the protocol, e.g. mysite.com */
-						__( 'You can access your backups on your site\'s <a href="https://wordpress.com/activity-log/%s" target="_blank" rel="noopener noreferrer">Activity</a> page.', 'jetpack' ),
+						/* Translators: first variable is the full URL to the new dashboard */
+						__( 'You can access your backups at this <a href="%s" target="_blank" rel="noopener noreferrer">new dashboard</a>.', 'jetpack' ),
```

Comment:
> new dashboard

"New" is tricky. Will this dashboard be new to first-time users? At what point will it not be new to existing users? 

Can we update this to something more future-proof?

Reply:
> > new dashboard
> 
> "New" is tricky. Will this dashboard be new to first-time users? At what point will it not be new to existing users?
> 
> Can we update this to something more future-proof?

Agree, I'm not in love with the verbiage here. Perhaps we drop the word `new` and instead hyperlink `this dashboard`.